app/models/user.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable require-atomic-updates */
+const bcrypt = require('bcrypt');
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        validate: {
+          notNull: {
+            args: true,
+            msg: 'Name cannot be null'
+          },
+          notEmpty: {
+            args: true,
+            msg: 'Name cannot be empty'
+          }
+        }
+      },
+      last_name: { type: DataTypes.STRING },
```

Comment:
camelCase in the models. snake_case in the migrations.

Reply:
done :)